src/plugins/discover/public/application/components/sidebar/discover_field.tsx

```diff
@@ -195,6 +221,36 @@ export function DiscoverField({
     );
   }
 
+  const shouldRenderMultiFields = !!multiFields;
+  const renderMultiFields = () => {
+    if (!multiFields) {
+      return null;
+    }
+    return (
+      <React.Fragment>
+        <EuiTitle size="xs">
+          <h5>
+            {i18n.translate('discover.fieldChooser.discoverField.multiFields', {
+              defaultMessage: 'Multi fields',
+            })}
+          </h5>
+        </EuiTitle>
+        {multiFields.map((entry) => (
+          <FieldButton
+            size="s"
+            className="dscSidebarItem"
```

Comment:
```suggestion
            className="dscSidebarItem dscSidebarItem-multi"
```

Add extra class to be able to target this type of element and remove padding-left.